sysbuild/image_configurations/firmware_loader_image_default.cmake

```diff
@@ -5,17 +5,6 @@
 #
 
 if(SB_CONFIG_BOOTLOADER_MCUBOOT)
```

Comment:
Which mechanism is responsible for passing MCUboot signing keys to the firmware loader image then?